airgun/views/report_template.py

```diff
@@ -95,6 +95,7 @@ class ReportTemplateGenerateView(BaseLoggedInView):
     email_to = TextInput(id='report_template_report_mail_to')
     inputs = TextInputsGroup(locator='.//form')
     submit = Text('//input[@name="commit"]')
+    generating = Text('//div[@title="Generating a report"]/span[2]')
```

Comment:
```suggestion
    generating = Text('//div[@id="template_generator"]//span[2]')
```
But I think you should be able to do with just `Text('//div[@id="template_generator"]')`